Bigger UI improvements

mmeldo opened the pull request:
- refactor of the board where some of the areas were changed from flex to grid
- refactor of the prompt area to status area
- player status row moved to bottom of new status area
- added icons for chat messages
- make the prompt panel draggable
- slight improvements in the location area styles

stopfstedt commented on Components/GameBoard/Messages.jsx:
```diff
@@ -17,7 +17,17 @@ class Messages extends React.Component {
+            'ghost rock': { className: 'icon-token', imageSrc: '/img/icons/ghostrock.png' },
+            influence: { className: 'icon-token', imageSrc: '/img/icons/influence.png' },
+            'control point': { className: 'icon-token', imageSrc: '/img/icons/control.png' },
+            'control points': { className: 'icon-token', imageSrc: '/img/icons/control.png' },
```
neither text strings `ghost rock`, `control point`, nor `control points` work for me in the text message window. they just render as plain text. all other, single word tokens are replaced by their icon equivalent and work as intended.

suggesting the addition of the `CP`  abbreviation for control point.

it would also be great if the icons in the message window would show their name on mouse-over. a `title` attribute on the respective `img` tag would do the trick.

mmeldo replied:
> neither text strings `ghost rock`, `control point`, nor `control points` work for me in the text message window. they just render as plain text. all other, single word tokens are replaced by their icon equivalent and work as intended.

Oh well, of course it does not work since the message is tokenized using the space separator. So I just removed those and updated existing messages to use either `GR` or `CP`.

> it would also be great if the icons in the message window would show their name on mouse-over. a title attribute on the respective img tag would do the trick.

Good idea, done.

stopfstedt reviewed (changes requested):
overall, this looks good to me. see my comment re: text-to-icon replacement in the chat window.

one more change though: 
the file `assets/img/icons/Influence.png` should be renamed to `assets/img/icons/influence.png`.

mmeldo commented:
Pushed additional changes:
- Improved @stopfstedt changes a little so only the card panels and piles are scrollable and player status area is fixed
- changed the prompt panel position to absolute so it does not affect shootout status area
- updated the shootout status area so it can be collapsed and makes the town area even bigger
- updated townsquare alignment so now its width is the same as the longest street. this needed to be done to fix situations when street was outside the viewport

I tested changes with some really small resolutions and it works quite good. Overall, I have to say I am very happy with how the board is now.
There are still some improvements planned (mostly just nicifications).

stopfstedt reviewed (approved):
LGTM
